buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -225,35 +656,44 @@
       "value": null,
       "isSecret": true
     },
-    "PB_VsoRepoUrl": {
-      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
-    },
     "PB_VsoAccountName": {
       "value": "dn-bot"
     },
     "PB_VsoPassword": {
       "value": null,
       "isSecret": true
     },
+    "PB_Branch": {
```

Comment:
> It's not the right value though. It doesn't get passed in correctly - so it will always be master even if you are on another branch.

That may be true, I haven't looked at what is actually being passed from PipeBuild recently.  I know that, for my own private builds, I quite often set PB_SourceBranch to a different value for validation.